### PR TITLE
feat(SDK): add job create sdk

### PR DESCRIPTION
**Pull request by tianweidut**
## Modules
- [x] Python-SDK

## Checklist
- [x] run code format and lint check
- [x] add unit test
- [ ] add necessary doc

### CODE REVIEW

File: client/starwhale/api/_impl/job/model.py [hunk 1]
```diff
@@ -226,3 +221,192 @@ def get_table_rows(
...
+        print(job.status)
+        ```
+        Returns:
+            Job Object
+        """
+        if isinstance(project, str):
+            project = Project(project)
+
+        if project.instance.is_local:
+            return cls.create_local(
```
Comment (goldenxinxing) [inline client/starwhale/api/_impl/job/model.py:305]: Are there loss the runtime parameter?

File: client/starwhale/core/model/model.py [hunk 2]
```diff
@@ -945,43 +948,80 @@ def diff(self, compare_uri: Resource) -> t.Dict[str, t.Any]:
...
-            handler=run_handler,
+            dev_mode=dev_mode,
+            dev_password=dev_mode_password,
         )
+
+        if ttl > 0:
+            # if want to live forever, do not set this field for server api
+            kwargs["time_to_live_in_sec"] = ttl
+
+        if run_handler is None:
```
Comment (goldenxinxing) [inline client/starwhale/core/model/model.py:1020]: Replace with 'if run_handler:'? so None or '' would be covered.

Comment (tianweidut) [reply client/starwhale/core/model/model.py:1020]: updated

**Review by goldenxinxing — APPROVED** [review]
LGTM

**Pull request merged**